src/napkin_multi_printer.mli

```diff
@@ -0,0 +1 @@
+val print: [ (* `refmt of string (* path to refmt *) | `ml *) `res ] -> input: string -> unit
```

Comment:
assuming this is the public interface, the return value should be string?

Reply:
still WIP, typo, should be string

Reply:
or it should be 
```
val print : ... -> input:string -> Format.formatter -> unit
```
whichever you like